.github/release.yaml

```diff
@@ -0,0 +1,12 @@
+changelog:
+  exclude:
+    labels: ['wontfix', 'question', 'duplicate', 'invalid']
+  categories:
+    - title: 'Enhancements'
+      labels: ['enhancement']
+    - title: 'Bug Fixes'
+      labels: ['bug']
+    - title: 'Documentation'
+      labels: ['documentation', 'docs']
+    - title: 'Dependency Upgrades'
+      labels: ['dependency-upgrade', 'dependencies']
```

Comment:
Looks like `dependency-upgrade` is not a label, maybe we should remove it to avoid confusion.